core-bundle/contao/dca/tl_templates.php

```diff
@@ -521,9 +523,56 @@ public function compareButton($row, $href, $label, $title, $icon, $attributes)
 	 */
 	public function dragFile($row, $href, $label, $title, $icon, $attributes)
 	{
+		if ($this->isTwigFile($row))
+		{
+			$attributes .= ' disabled="true"';
```

Comment:
why not just `return ''` here (as discussed)?

Reply:
Because then we would need to do it for all operations, not only those that aren't already disabled via the voters.

Reply:
(or the buttons would not align anymore).

Reply:
> (or the buttons would not align anymore).

Aren't we gonna hide them within the context menu 🙈

Reply:
```suggestion
			return Image::getHtml($icon, $label);
```

I know that there is a 1 pixel gap this way, but this will be fixed in a separate PR.